Source/Scene/Cesium3DTile.js

```diff
@@ -82,7 +82,20 @@ define([
         this._header = header;
         var contentHeader = header.content;
 
-        this._boundingVolume = createBoundingVolume(header.boundingVolume);
+        /**
+         * The local transform of this tile
+         * @type {Matrix4}
+         */
+        this.transformLocal = defined(header.transform) ? Matrix4.unpack(header.transform, 0) : Matrix4.clone(Matrix4.IDENTITY);
```

Comment:
`0` is the default.  `fromArray` (which is the same as `unpack`) may also be more clear in this context.